src/interaction/InteractionManager.js

```diff
@@ -341,6 +459,56 @@ InteractionManager.prototype.dispatchEvent = function ( displayObject, eventStri
 };
 
 /**
+ * If pointer events aren't available on a device, this will turn either the touch or mouse events into pointer events
+ * This allows a developer to just listen for emitted pointer events on interactive sprites
+ *
+ * @param eventString {string} the name of the event (e.g, mousedown)
+ * @private
+ */
+InteractionManager.prototype.normalizeEventString = function ( eventString )
+{
+    if (!this.supportsPointerEvents)
+    {
+        if (this.supportsTouchEvents && Device.any)
```

Comment:
is Device.any needed? I would think that anything that is outputting touch events you'd want to convert to pointer events, even if it isn't a recognized Android or iOS device. That would also allow you to turn this function into a straight dictionary check instead of switch statements.